### DISCUSSION TITLE
In an N1QL WHERE clause, how to check if a field value is in a List

### DISCUSSION THREAD

**Discussion by natgross** [cofu-app/cbl-dart#498] [Q&A]
Say, all my documents have a String field named "myField" ({"myField":'any string here'}). In the class where I'm running the query, I have a list of Strings List<String> myList.
I am trying to write `'WHERE myField IN $myList'` but it's being kicked out due to the [brackets]. Seems like a simple thing, but no can do.
thanks.

**Comment by blaugold** [comment]
SQL++ (previously N1QL) requires a list of expressions as the second argument to the `IN` operator: `expression IN (value1, value2, .... value_n)`.
